docs/pages/installation/_index.md

```diff
@@ -5,20 +5,252 @@ weight: 2
+{{% /notice %}}
+
+{{% /tab %}}
+{{% tab name="GKE" %}}
+
+##### Supported CNIs 
+
+Liqo supports GKE clusters using the default CNI:
+* [Google GKE - VPC-Native](https://cloud.google.com/kubernetes-engine/docs/how-to/alias-ips)
+
+{{% notice note %}}
+Liqo does not support GKE Autopilot Clusters
+{{% /notice %}}
+
+##### Configuration
+
+To install Liqo on GKE, you should at first create a service account for liqoctl, granting the read rights for the GKE clusters (you may reduce the scope to a specific cluster if you prefer).
+
+First, let's start exporting required variables:
+```bash
+export SERVICE_ACCOUNT_ID=liqoctl-sa #the name of the service account used to interact by liqoctl with GCP
+export PROJECT_ID=XYZ # the id of the cluster project
+export SERVICE_ACCOUNT_PATH=~/.liqo/gcp_service_account # the path where the google service account is stored
+export GKE_CLUSTER_ZONE=europe-west-1b # the GCP zone where your GKE cluster is executed
+```
+
+First, you should create a GCP Service account. 
+This will provide you an identity used by Liqoctl to query all the information needed to properly configure Liqo on your cluster.
+
+The ServiceAccount can be created using:
+```bash
+gcloud iam service-accounts create ${SERVICE_ACCOUNT_ID} \
+    --description="DESCRIPTION" \
+    --display-name="DISPLAY_NAME" \
+    --project="${PROJECT_ID}"
+```
+
+Second, you should provide the ServiceAccount just created with the rights to inspect the cluster and virtual networks parameters:
+
+```bash
+gcloud projects add-iam-policy-binding ${PROJECT_ID} \
+    --member="serviceAccount:${SERVICE_ACCOUNT_ID}@${PROJECT_ID}.iam.gserviceaccount.com" \
+    --role="roles/container.clusterViewer"
+gcloud projects add-iam-policy-binding ${PROJECT_ID} \
+    --member="serviceAccount:${SERVICE_ACCOUNT_ID}@${PROJECT_ID}.iam.gserviceaccount.com" \
+    --role="roles/compute.networkViewer"
+```
+
+Third, you should create and download valid service accounts keys, as presented [by the official documentation](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys).
+
+The keys will be used by liqoctl to authenticate to GCP as the service account we just created.
+
+```bash
+gcloud iam service-accounts keys create ${SERVICE_ACCOUNT_PATH} \
+    --iam-account=${SERVICE_ACCOUNT_ID}@${PROJECT_ID}.iam.gserviceaccount.com
+```
+
+{{% /tab %}}
+{{% tab name="K3s" %}}
+
+##### Supported CNIs
+
+Liqo supports K3s clusters using the following CNIs:
+* [Weave](https://github.com/weaveworks/weave)
+* [Flannel](https://github.com/coreos/flannel)
+* [Canal](https://docs.projectcalico.org/getting-started/kubernetes/flannel/flannel)
+* [Calico](https://www.projectcalico.org/)
+
+##### Configuration
+
+**Optional**: You only have to export the KUBECONFIG environment variable.
+Otherwise, liqoctl will try to use the kubeconfig in kubectl default path (i.e. `${HOME}/.kube/config` )
```

Comment:
```suggestion
You only have to export the KUBECONFIG environment variable.
Otherwise, liqoctl will use the kubeconfig in kubectl default path (i.e. `${HOME}/.kube/config` )
```

For consistency with kubeadm